### ISSUE TITLE
Issue with Image size 3024x4032

### ISSUE THREAD

**Issue by RobertCalamari** [oliver-moran/jimp#736]
# Expected Behavior

I am just trying to take an image from one folder and move it to another folder.

# Current Behavior

However, I am having an issue with images that are size 3024x4032. When my code is ran and I check for the width and the height, it looks as if they switched. Code is below.

## Steps to Reproduce

The code takes 5 images named "pics (_n_).jpg" and moves them to a new folder. 

function runchanger(num, callback){
    if(num == 0){
        console.log("DONE");
    }
    else{
        var filetoget = './client/img/imgprcss/tochange/pics (' + num + ').jpg';
         var filechanged = './client/img/imgprcss/changed/keeping (' + num + ').jpg';

         Jimp.read(filetoget, (err, image) => {
              if (err) throw err;
              console.log(num + " - were her changing the image - width:" + image.bitmap.width + " | height: " + image.bitmap.height);
                image.write(filechanged);
                callback(num-1, runchanger);
        });     
        
    }
    
}
     
runchanger(5, runchanger);

IF YOUR ISSUE DEPENDS ON A PARTICULAR IMAGE BE SURE TO INCLUDE THIS AS WELL. WE CAN'T REPORDUCE IF WE DON'T HAVE YOUR IMAGE

Images are attached. I included some that are fine to show that not all of them are effected. 
Here are the images before processing:
![pics (396)](https://user-images.githubusercontent.com/36252593/57721391-0da7b000-7652-11e9-9506-91629bcb1ee9.JPG)
![pics (394)](https://user-images.githubusercontent.com/36252593/57721329-e7821000-7651-11e9-960b-6c7540d5f116.JPG)
![pics (395)](https://user-images.githubusercontent.com/36252593/57721330-e7821000-7651-11e9-8510-67d581bb4238.JPG)
![pics (399)](https://user-images.githubusercontent.com/36252593/57721331-e81aa680-7651-11e9-8b04-68541cbe55d8.JPG)
![pics (400)](https://user-images.githubusercontent.com/36252593/57721332-e81aa680-7651-11e9-933f-cb9272edb86d.JPG)

Here are the images after:
![keeping (395)](https://user-images.githubusercontent.com/36252593/57721410-1c8e6280-7652-11e9-89ce-e8a8e0c0173f.jpg)
![keeping (396)](https://user-images.githubusercontent.com/36252593/57721411-1c8e6280-7652-11e9-8900-115781c213cb.jpg)
![keeping (399)](https://user-images.githubusercontent.com/36252593/57721412-1c8e6280-7652-11e9-8d91-f1054ce19a73.jpg)
![keeping (400)](https://user-images.githubusercontent.com/36252593/57721413-1c8e6280-7652-11e9-814c-c16e04b6904f.jpg)

## Context
Running the code through node.js

- Jimp Version: 0.6.4
- Operating System: Windows 10
- Node version: v8.12.0

**Comment by RobertCalamari** [comment]
I notice here that when I posted the files in here they are sideways which would make sense why it is saying that the width is longer. However this seems odd since the images in the windows explorer are vertical:
Images before processing. I made the middle photo horizontal to see if that would change it:
![image](https://user-images.githubusercontent.com/36252593/57721701-d685ce80-7652-11e9-9e52-a688c0582f55.png)

Images after processing:
![image](https://user-images.githubusercontent.com/36252593/57721757-f917e780-7652-11e9-8853-00af8cc52350.png)

What is confusing is that when you look at the properties of the image, the height and width change when I rotate it.
![image](https://user-images.githubusercontent.com/36252593/57721841-2d8ba380-7653-11e9-8f6c-59148a53b959.png)
